### PR TITLE
UPPSF-176 Write ISO 3166-1 Code to Locations

### PR DESCRIPTION
**Pull request by karsov**

### CODE REVIEW

**Comment by emildimitrov** [comment]
@karsov if we'r going to address only the write part here maybe it makes sense to make another jira ticket for the read part, because there is tasks which depend on this task (the unit tests for concordances-api)

**Comment by karsov** [comment]
> r

These changes also allow reads of the ISO code.

File: concepts/concepts_service.go [hunk 1]
```diff
@@ -161,6 +164,8 @@ type neoConcept struct {
 	YearFounded            int      `json:"yearFounded,omitempty"`
 	LeiCode                string   `json:"leiCode,omitempty"`
 	ParentOrganisation     string   `json:"parentOrganisation,omitempty"`
+	// Location
+	ISO31661 string `json:"iso31661,omitempty"`
```
Comment (epavlova) [inline concepts/concepts_service.go]: I know that in this case  not all of the struct fields are aligned before the PR. However they should be. When I'm saying aligned means the that the fields names, types and metadata should start on the same vertical line of all the fields. So keep it in mind the next time.

File: concepts/model.go [hunk 2]
```diff
@@ -91,6 +93,8 @@ type Concept struct {
 	LeiCode                string   `json:"leiCode,omitempty"`
 	ParentOrganisation     string   `json:"parentOrganisation,omitempty"`
 	IsDeprecated           bool     `json:"isDeprecated,omitempty"`
+	// Location
+	ISO31661 string `json:"iso31661,omitempty"`
```
Comment (epavlova) [inline concepts/model.go]: The same alignment issue.

File: concepts/model.go [hunk 3]
```diff
@@ -43,6 +43,8 @@ type AggregatedConcept struct {
 	YearFounded            int      `json:"yearFounded,omitempty"`
 	LeiCode                string   `json:"leiCode,omitempty"`
 	IsDeprecated           bool     `json:"isDeprecated,omitempty"`
+	// Location
+	ISO31661 string `json:"iso31661,omitempty"`
```
Comment (epavlova) [inline concepts/model.go]: The same alignment issue

**Pull request merged**